File: 0242-valid-anagram/0242-valid-anagram.cpp
#include <bits/stdc++.h>
class Solution {
public:
    bool isAnagram(string s, string t) {
        vector<char> a;
        vector<char> b;
        for(char c:s){
            a.push_back(c);
        }
        for(char d: t){
            b.push_back(d);
        }
        sort(a.begin(),a.end());
        sort(b.begin(),b.end());
        return a==b;
    }
};

File: 0242-valid-anagram/0242-valid-anagram.py
class Solution:
    def isAnagram(self, s: str, t: str) -> bool:
        if len(s)!=len(t): return False
        a={}
        for i in s:
            a[i]=a.get(i,0)+1
        for i in t:
            if i not in a or a[i]==0:
                return False
            a[i]-=1
        return True